### PR TITLE
Add new packs for TCU and WOS, as well as the 'real' Carolyn Fern

### PR DESCRIPTION
**Pull request by zzorba**
Changed books.json to refer to the 'original' cards (like
Jenny/Roland), while the new Carolyn refers to both. Not sure if this
will require some kind of backfill to merge the Carolyn over to the
canonical non-promo one?

### CODE REVIEW

File: pack/tcu/tcu.json [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "Cut to the heart of the matter.",
+		"illustrator": "Chris Peuler",
+		"name": "Ace of Swords",
+		"pack_code": "tcu",
+		"position": 23,
+		"quantity": 2,
+		"subname": "Let Your Arrows Fly True",
+		"text": "You get +1 [combat].]\n[reaction] When the game begins, if Ace of Swords is in your opening hand: Put it into play.",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: After [combat] there's an extra ]...

File: pack/tcu/tcu.json [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+		"slot": "Tarot",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Please can 'slot' come after 'quantity' for alphabetical reasons!

File: pack/tcu/tcu.json [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "Let go, and embrace a new truth.",
+		"illustrator": "Chris Peuler",
+		"name": "Death • XIII",
+		"pack_code": "tcu",
+		"position": 27,
+		"quantity": 2,
+		"subname": "Free from the Past",
+		"text": "You get +1 [intellect].]\n[reaction] When the game begins, if Death • XIII is in your opening hand: Put it into play.",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Same issue with rogue bonus ], and with slot.

File: pack/tcu/tcu.json [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+		"slot": "Tarot",
+		"traits": "Tarot.",
+		"type_code": "asset",
+		"xp": 1
+	},
+	{
+		"code": "05027",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "guardian",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Seeker, please.

File: pack/tcu/tcu.json [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+		"faction_code": "rogue",
+		"flavor": "Open your eyes, and listen to your shadow.",
+		"illustrator": "Chris Peuler",
+		"name": "The Moon • XVIII",
+		"pack_code": "tcu",
+		"position": 31,
+		"quantity": 2,
+		"subname": "Message from Your Inner Self",
+		"text": "You get +1 [agility].\n[reaction] When the game begins, if The Moon • XVIII is in your opening hand: Put it into play.",
+		"slot": "Tarot",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Same thing on slot position.

File: pack/tcu/tcu.json [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+		"faction_code": "survivor",
+		"flavor": "It is time to rise up again.",
+		"illustrator": "Chris Peuler",
+		"name": "Five of Pentacles",
+		"pack_code": "tcu",
+		"position": 39,
+		"quantity": 2,
+		"subname": "From the Brink",
+		"text": "You get +1 health and +1 sanity.\n[reaction] When the game begins, if Five of Pentacles is in your opening hand: Put it into play.",
+		"slot": "Tarot",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Alphabet!

File: pack/tcu/tcu.json [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+		"traits": "Tarot.",
+		"type_code": "asset",
+		"xp": 1
+	},
+	{
+		"code": "05040",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "neutral",
+		"flavor": "It is time to rise up again.",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: No flavor on Ace of Rods

File: pack/tcu/tcu.json [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+		"illustrator": "Robert Lackey",
+		"name": "Ace of Rods",
+		"pack_code": "tcu",
+		"position": 40,
+		"quantity": 2,
+		"subname": "The Fateful Step",
+		"text": "[fast] During your turn, remove Ace of Rods from the game: You may take an additional action this turn, during which you get +2 shroud to each of your skills.\n[reaction] When the game begins, if Ace of Rods is in your opening hand: Put it into play.",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: You've added the word 'shroud' in the middle here, after '+2'!

File: pack/tcu/tcu.json [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+		"slot": "Tarot",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Alpha.

File: pack/tcu/tcu.json [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+		"flavor": "It is time to rise up again.",
+		"illustrator": "Robert Lackey",
+		"name": "The Tower • XVI",
+		"pack_code": "tcu",
+		"position": 42,
+		"quantity": 2,
+		"subname": "Circumstances Beyond Your Control",
+		"subtype_code": "basicweakness",
+		"text": "You cannot commit cards to skill tests while The Tower • XVI is in your hand.\nIf The Tower • XVI is drawn in your opening hand during setup <i>(before or after taking a mulligan)</i>, you cannot replace it. It must stay in your opening hand.",
+		"slot": "Tarot",
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json]: Alpha

File: pack/tcu/tcu.json (hunk 8 above)
Comment (zzorba) [reply pack/tcu/tcu.json]: lol, where did that come from

**Comment by zzorba** [comment]
All fixed, thanks for the review!

**Pull request merged**